lib/shellhandlerapt.py

```diff
@@ -56,7 +56,7 @@ def _check_versions(self, package, update=True):
             if installed == "(none)":
                 installed = False
             if candidate == "(none)":
-                candiate = False
+                candidate = False
```

Comment:
This is a code change. The old code would probably have done effectively nothing...